third_party/multiarch/qemu-user-static/register/register.sh

```diff
@@ -19,13 +19,13 @@ entries="aarch64 aarch64_be alpha arm armeb hppa m68k microblaze microblazeel mi
 if [ "${1}" = "--reset" ]; then
     shift
     (
-    cd /proc/sys/fs/binfmt_misc
+    cd /proc/sys/fs/binfmt_misc || exit
```

Comment:
do we normally make edits like this under third_party?

Reply:
for this one at least, yes https://github.com/kubernetes/kubernetes/pull/69820

Reply:
for the go code we usually have edited code under third_party/forked, but I don't think we've been fully consistent about things being in `third_party/forked` vs `third_party` ...